### ISSUE TITLE
Failed to parse SourceMap: http://localhost:3000/packages/undefined

### ISSUE THREAD

**Issue by tomitrescak** [gadicc/meteor-react-hotloader#21]
Hi, am getting this error "Failed to parse SourceMap: http://localhost:3000/packages/undefined" seems that it does not affect anything, but I see it in my console. Any idea? Changing to original ecmascript resolves the problem.

**Comment by clayne11** [comment]
I'm seeing this as well.

**Comment by tomitrescak** [comment]
It seems that it just does not load sourcemaps at all. When I have a sourcemap in place it says following (e.g. `Failed to parse SourceMap: http://localhost:3000/home_view.jsx.map`)

**Comment by gadicc** [comment]
Thanks for reporting.

The package doesn't support source maps (yet), but this should only affect the code that comes through on an update (as `hot.js`).

I realized though that I broke the source map for module-runtime.js on the latest release, and this file is involved in the all the module loading, so could be related.  Could you see if those errors are still present on the older `v1.3.0-1`?

A few more Q's:

1. What browser is this on?  Since I don't get those errors my version of Chrome.

1. When you say you have a source map in place, you mean through Meteor?  I know also that there have been reports of some broken source maps in Meteor 1.3; does this 2nd error resolve itself with the regular `ecmascript` package?

**Comment by gadicc** [comment]
Don't worry about the older version, I just released a new version which reverts this behaviour, `gadicc:ecmascript-hot@1.3.0_4`.  Could you please just reconfirm that one or both of these errors are still present?

**Comment by gadicc** [comment]
Here's e.g. some more info on sourcemap problems in Meteor 1.3 generally:

https://forums.meteor.com/t/debugging-on-chrome-with-meteor-1-3-modules-and-angular/20433

**Comment by sammkj** [comment]
I can confirm that this issue still persists in the new release. 

PS: I'm using Chrome v49.0.2623.110.

**Comment by gadicc** [comment]
Hey guys, I can't seem to reproduce this on Chrome 48-50 on Linux, on a few different apps.  Can someone volunteer a repo that I can clone where we know for a fact the problem exists?

@tomitrescak can you also confirm that that 2nd error occurs with the original `ecmascript` too, and isn't related to the source map problems in Meteor 1.3?

**Comment by tomitrescak** [comment]
The issue is persitent on my project as well ;/ unfortunatelly I cannot confirm the issue with ecmascript as I depend on your babel plugin loading since I use jsx-control-statements

**Comment by proxiper** [comment]
I have the same problem.

I've created demo repo https://github.com/proxiper/meteor-ecmascript-hot-source-map-problem-demo

The error is: "Failed to parse SourceMap: http://localhost:3000/packages/undefined".

I use Chrome 49.0.2623.110 (64-bit) on MacOS 10.11.2 and Windows 8.

**Comment by lyquocnam** [comment]
me too on windows 10 x64

**Comment by gadicc** [comment]
Thanks guys (and @proxiper for the repo), I finally figured out the problem!  I think it's fixed now but it's hard to test... I'd love if people could try the pre-release I just mentioned in https://github.com/gadicc/meteor-react-hotloader/issues/26#issuecomment-205866733 and confirm.

The reason it was so hard for me to reproduce was because it was a caching issue... finally ran into it when going back to a different old project today.  Looks solved now but now that I've cleared the cache I have no way to check :/
